codemeta.json

```diff
@@ -1,36 +1,17 @@
 {
-  "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
-  "@type": "SoftwareSourceCode",
-  "name": "PlasmaPy",
-  "description": "An open source software package for plasma research and education",
-  "keywords": [
-    "plasma physics",
-    "open source",
-    "scientific software",
-    "Python"
-  ],
-  "version": "0.3.0",
-  "license": "https://github.com/PlasmaPy/PlasmaPy/blob/master/LICENSE.md",
-  "creator": [
-    {
-      "@type": "Organization",
-      "name": "PlasmaPy Community"
-    },
-    {}
-  ],
-  "codeRepository": "https://github.com/PlasmaPy/PlasmaPy",
-  "programmingLanguage": [
-    "Python 3.6",
-    "Python 3.7"
-  ],
-  "copyrightHolder": "PlasmaPy Developers",
-  "isAccessibleForFree": true,
-  "issueTracker": "https://github.com/PlasmaPy/plasmapy/issues",
-  "readme": "https://github.com/PlasmaPy/PlasmaPy/blob/master/README.md",
-  "developmentStatus": "WIP",
-  "buildInstructions": "http://docs.plasmapy.org/en/stable/install.html",
-  "referencePublication": "https://doi.org/10.5281/zenodo.1238132",
-  "funder": [
+    "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
+    "@type": "SoftwareSourceCode",
+    "license": "https://spdx.org/licenses/BSD-2-Clause-Patent",
+    "codeRepository": "https://github.com/PlasmaPy/plasmapy",
+    "contIntegration": "https://dev.azure.com/plasmapy/PlasmaPy/_build/results?buildId=1733&view=results",
```

Comment:
Do we want this to point to that particular build (PR 900)?

Reply:
And this, my friend, is why we do code review!